samples/client/others/typescript-axios/with-separate-models-and-api-inheritance/common.ts

```diff
@@ -85,6 +85,7 @@ export const setOAuthToObject = async function (object: any, name: string, scope
 }
 
 function setFlattenedQueryParams(urlSearchParams: URLSearchParams, parameter: any, key: string = ""): void {
+    if (parameter == null) return;
```

Comment:
where does this come from?

Reply:
I've also noticed this. I just ran the commands mentioned in the PR template:
```sh
./mvnw clean package 
./bin/generate-samples.sh
./bin/utils/export_docs_generators.sh
```